demos/APS Monte Carlo Example.ipynb

```diff
@@ -287,7 +287,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.7.6"
```

Comment:
does it make sense to just have a single Monte Carlo notebook (i.e. the new one), or do we need 2 separate ones?

Reply:
I would say it's good to have one for each type, since they can watch the whole chain. Alternatively we could bundle it all in one with only one case uncommented. We should discuss in the meeting today which way people think the documentation would be more clear.

Reply:
Opening an issue for collapsing these down to one...